### PR TITLE
#67 fix query and add response field

### PR DESCRIPTION
**Pull request by setiadijoe**
- fix query get list
- add field category_id, kabkota_id, kec_id, kel_id

### CODE REVIEW

**Review by firmanJS — APPROVED** [review]

**Comment by firmanJS** [comment]
LGTM !

**Review by rindibudiaramdhan — APPROVED** [review]

File: repository/mysql/mysql.go [hunk 1]
```diff
@@ -38,8 +39,7 @@ func (r *PhonebookRepository) GetListPhoneBook(ctx context.Context, params *mode
 	query.WriteString(selectQuery.String())
 
 	if params.SortBy != nil && params.OrderBy != nil {
-		query.WriteString(" ORDER BY ? ? ")
-		queryParams = append(queryParams, helper.GetStringFromPointer(params.SortBy), helper.GetStringFromPointer(params.OrderBy))
+		query.WriteString(fmt.Sprintf(" ORDER BY %s %s ", helper.GetStringFromPointer(params.SortBy), helper.GetStringFromPointer(params.OrderBy)))
```
Comment (sandisunandar99) [inline repository/mysql/mysql.go:42]: #ask
this line can you make more readable code maybe? 😄

Comment (setiadijoe) [reply repository/mysql/mysql.go:42]: I used pointer on every field parameters, so I need to check if the params is nil and to get the value, I used helper get value from pointer
I use string formatter to input to the query
just look on green line

Comment (sandisunandar99) [reply repository/mysql/mysql.go:42]: if using parameter like this
`param1 := helper.GetStringFromPointer(params.SortBy)`
`param2 := helper.GetStringFromPointer(params.OrderBy)`

what effect?

**Pull request merged**